frigate/api/media.py

```diff
@@ -133,6 +133,15 @@ def latest_frame(
         "regions": params.regions,
     }
     quality = params.quality
+    mime_type = extension
```

Comment:
I don't think there is a need for this. jpg is supported https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types